person.rb

```diff
@@ -29,4 +34,8 @@ def of_age?
   def can_use_servicies?
     of_age? || @parent_permission
   end
+
+  def add_rental(rental)
+    @rentals.push(rental) unless @rentals.include?(rental)
+  end
```

Comment:
- [ ] This method should take book and date as parameters and create a new instance of rental achieving the 'has-many' association
```
 def add_rental(book, date)
    Rental.new(date, self, book)
  end
```

Reply:
Thanks @VICTOR-CHUKWU! I have applied the required changes